src/components/Button/index.js

```diff
@@ -52,26 +57,22 @@ export default class Button extends PureComponent {
       className,
       fullWidth,
       kind,
-      link,
       loading,
       rounded,
-      secondary,
```

Comment:
Blaming @jstnjns when we forget to update these in the main masterclass repo before versioning :X

Reply:
i'll allow it